src/makeInfinityAdder.js

```diff
@@ -7,6 +7,26 @@
 
 function makeInfinityAdder() {
   // write code here
+  const arr = [];
```

Comment:
You have a nice solution, yet you can simplify your code and avoid array with reduce methods
```suggestion
let result = 0;
```